# empty request , node listens to required topics on its own
int32 EMPTY = 1
int32 FULL = 2
int32 call

---

# Return with the point of action execution
geometry_msgs/PointStamped action_location

# Whether feature extraction failed or succeeded
int32 FAILURE = 1
int32 SUCCESS = 2
int32 result
